### PR TITLE
Removed Loggings & Added GitHub Workflows

### PR DESCRIPTION
**Pull request by saeidalz13**
# Overview

- Removed the unnecessary loggings
- Added `ci.yml` for CI/CD.

### CODE REVIEW

File: models/websocket_models.go [hunk 1]
```diff
@@ -31,7 +29,7 @@ const (
 	CodeAttack
 	CodeEndGame
 	CodeInvalidSignal
-	CodeSignalAbsent  // if the req msg does not contain "code" field
+	CodeSignalAbsent // if the req msg does not contain "code" field
```
Comment (mori-ahk) [inline models/websocket_models.go:32]: Why do we need this?

File: api/handlers_api.go [hunk 2]
```diff
@@ -9,6 +9,8 @@ import (
 	md "github.com/saeidalz13/battleship-backend/models"
 )
 
+// TODO: Remove unnecessary loggings
```
Comment (mori-ahk) [inline api/handlers_api.go:12]: Let's remove all the TODOs comments

File: models/websocket_models.go (hunk 1 above)
Comment (saeidalz13) [reply models/websocket_models.go:32]: Any payload that doesn't have `Code` in its JSON is invalid and we catch that edge case with that

File: api/handlers_api.go (hunk 2 above)
Comment (saeidalz13) [reply api/handlers_api.go:12]: Aight

**Comment by saeidalz13** [comment]
ci.yml file needs to be fixed. I need to set up API token for fly.
Deploy fails for now

**Comment by saeidalz13** [comment]
Deploy is fixed now

File: models/websocket_models.go (hunk 1 above)
Comment (mori-ahk) [reply models/websocket_models.go:32]: yeah I got that, it's just that this is not the right PR for this change. Not blocking though just wanted to flag that for future PRs

**Review by mori-ahk — APPROVED** [review]

**Pull request merged**